cloudnet/cloudnet.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "CloudNet3",
+  "display": "CloudNet 3",
+  "type": "cloudnet",
+  "install": [
+    {
+      "files": [
+        "https://cloudnetservice.eu/cloudnet/version/release/${version}/CloudNet.zip"
+      ],
+      "type": "download"
+    },
+    {
+      "commands": [
+        "unzip CloudNet.zip",
+        "chmod -R 777 ."
```

Comment:
777 on an entire folder is bad
Only files which need some special perm should be handled. I would assume chmod +x start.sh

Also, the start.sh is bad for us. It runs screen, which is not what we want. The command should be ripped out, and added appropriately.

Also add java as a requirement. Refer to Minecraft templates.